var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -79,8 +91,7 @@ def variant_to_cfg(variant):
 
         with open("setup.cfg", "a") as setup:
             setup.write("[build_ext]\n")
-            variants = list(self.spec.variants)
```

Comment:
Does this not work anymore? For packages like GDAL, it's going to be a nightmare to list all variants twice, especially since the variants are conditional on version.

Reply:
No, this doesn't work since there's a new variant added everywhere: `buildsystem`. It was kind of hard for me to track down the error, since it seemed to be a wrong option passed during build. The option was something like:
```
buildsystems = 1
```
After some debugging I arrived at the logic here and explicitly listed the variants used previously. We could also do the other way around in other packages, and filter the variants that ought not to be used.